dockers/docker-fpm-frr/total_portstat.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
why is file in frr docker? can we put it into the sonic-utilities and then add unit test?

Reply:
Because I made a mistake and put it into the Dockerfile, and base_image_files.
I wanted to put it into the base image.
Now I remove the file completely
I wanted to use this short utility as a shortcut, because we need it for 201911. After that I wanted to implement this change as part of portstat.